recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-graphics/rcar-proprietary-graphic/prepare-graphic-package.bb

```diff
@@ -0,0 +1,18 @@
+LICENSE = "CLOSED"
+
+do_populate_lic[noexec] = "1"
+do_compile[noexec] = "1"
+do_install[depends] += "kernel-module-gles:do_install"
+do_install[depends] += "gles-module-egl-headers:do_install"
+do_install[depends] += "gles-user-module:do_install"
+
+do_install () {
+    if [ -d ${DEPLOY_DIR_IMAGE}/xt-rcar ]; then
```

Comment:
Hm, looks you already have such recipe...

Reply:
Done